### PR TITLE
DO-64 - phpunit

### PR DESCRIPTION
**Pull request by M4tini**
https://myparcelcombv.atlassian.net/browse/DO-64
- Updated phpunit and reverted it, because we need to support php 5.6 and thus run tests on that container.
- Code formatting.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "php-http/httplug": "^1.1"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
+    "phpunit/phpunit": "^7.5",
```
Comment (denniskoster) [inline composer.json]: I believe we went to great lengths to be able to still support php 5. Now that we require `phpunit 7.5` I believe we automatically require `php: ^7.1`

Comment (M4tini) [reply composer.json]: I did not alter `"php": ">=5.6"` so this only affects development.

However, I see that CircleCI is now running 4 tests on `php72` so with this update we cannot test on the php 5.6 container anymore. Let's discuss on monday.

Comment (M4tini) [reply composer.json]: I reverted the phpunit dependency.

File: src/Collection/RequestCollection.php [hunk 2]
```diff
@@ -76,8 +76,7 @@ public function get()
             $this->retrieveResources($firstPage);
         }
 
-        if (
-            $firstPage !== $secondPage
+        if ($firstPage !== $secondPage
```
Comment (denniskoster) [inline src/Collection/RequestCollection.php]: PSR-12 describes that, when splitting statements across multiple lines, the first statement must be on the next line. So we should probably revert this back. https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#51-if-elseif-else

Comment (M4tini) [reply src/Collection/RequestCollection.php]: Done

**Pull request merged**